app/models/profile.rb

```diff
@@ -3,6 +3,7 @@ class Profile < ApplicationRecord
 
   validates :data, presence: true
   validates :user_id, uniqueness: true
+  validates :location, :website_url, length: { maximum: 100 }
```

Comment:
Since these are normal columns now we can validate them here. `summary` is still validate via the `ProfileValidator` since it has some special cases for users with previously too long summaries.